eventsourced-nats/src/lib.rs

```diff
@@ -22,7 +21,7 @@ use thiserror::Error;
 pub enum Error {
     /// The connection to the NATS server cannot be established.
     #[error("Cannot connect to NATS server")]
-    Connect(#[from] std::io::Error),
```

Comment:
This seemed to cause a compilation error for me when running clippy although I'm not sure why this would have changed. There were a few other issues which can be explained by updates to clippy, not sure about this one though!

Reply:
Maybe NATS was updated. They have had some weird error handling and were planing some changes.